src/trio/_core/_run.py

```diff
@@ -1128,10 +1128,12 @@ def aborted(raise_cancel: _core.RaiseCancelT) -> Abort:
             try:
                 if not self._strict_exception_groups and len(self._pending_excs) == 1:
                     return self._pending_excs[0]
-                return BaseExceptionGroup(
-                    "collapsible" if not self._strict_exception_groups else "TODO",
-                    self._pending_excs,
+                exception = BaseExceptionGroup(
+                    "Exceptions from Trio nursery", self._pending_excs
                 )
+                if not self._strict_exception_groups:
+                    exception.add_note("collapsible")
+                return exception
```

Comment:
I thought `add_note` was 3.11+? Or does the `exceptiongroup` package have add_note implemented?

Reply:
Yep, [it's in the backport](https://github.com/agronholm/exceptiongroup/blob/c741f0b81c56d3b7a75b6096859e9e1463e02bbb/src/exceptiongroup/_exceptions.py#L87), so one way or another this is definitely valid.

Reply:
Seems kinda odd to be adding a note like this, if the user's able to see it it'll be rather confusing.

Reply:
If it's in the backport everything should be fine, just wanted to be sure!

Reply:
Another way to tag them would be to use a `WeakKeyDict` (or `WeakSet` for this case), so it's not even visible outside our code.

Reply:
I think it is good for it to be visible to end users though, if we plan on deprecating `strict_exception_groups=False`. With a custom exception it additionally makes it easy to add a deprecation warning (although it's maybe easier to just do that where they specify `strict_exception_groups=False`).
And e.g. if somebody is only catching single exceptions with loose groups, and then it crashes with a group, giving more context on that is great.